perfmetrics/scripts/ml_tests/checkpoint/Jax/run_checkpoints.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Fail on any error.
+set -e
+set -x
+echo "Running JAX checkpoint tests"
+
+sudo apt-get update
+# Install Git.
+echo "Installing git"
+sudo apt-get install git
+# Install Golang.
+#wget -O go_tar.tar.gz https://go.dev/dl/go1.24.0.linux-amd64.tar.gz -q
+architecture=$(dpkg --print-architecture)
+wget -O go_tar.tar.gz https://go.dev/dl/go1.24.0.linux-${architecture}.tar.gz -q
+sudo rm -rf /usr/local/go && sudo tar -C /usr/local -xzf go_tar.tar.gz
+export PATH=$PATH:/usr/local/go/bin
+# Install latest gcloud version for compatability with HNS bucket.
+function upgrade_gcloud_version() {
```

Comment:
Not blocking this PR. but this function is being used in many places. We can make separate script for this.